Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LtpValidationResult.java

```diff
@@ -0,0 +1,135 @@
+     * 
+     * @param generalState the success or failure state of the validation for the file or image
+     * @param generalErrors any general errors that occurred during validation
+     * @param conditionResults the success or failure state of each validation condition
+     * @param generalMessages any additional message provided by file-specific validation backend
+     */
+    public LtpValidationResult(
+        LtpValidationResultState generalState, 
+        List<LtpValidationError> generalErrors, 
+        List<LtpValidationConditionResult> conditionResults, 
+        List<String> generalMessages
+    ) {
+        this.state = generalState;
+        this.errors = generalErrors;
+        this.conditionResults = conditionResults;
+        this.additionalMessages = generalMessages;
+    }
+    
+    /**
+     * Return the overall validation success or failure state.
+     *
+     * @return the success or failure state of the validation
+     */
+    public LtpValidationResultState getState() {
+        return state;
+    }
+
+    /**
+     * Return general errors that occurred during validation.
+     * 
+     * @return a list of general validation errors
+     */
+    public List<LtpValidationError> getErrors() {
+        return errors;
+    }
+
+    /**
+     * Return the validation state (valid, warning, error) of each validation condition that was checked.
+     * 
+     * @return the validation state of each validation condition
+     */
+    public List<LtpValidationConditionResult> getConditionResults() {
+        return conditionResults;
+    }
+
+    /**
+     * Return a list of general validation messages that might provide additional information about
+     * why validation failed. Messages are not related to specific validation conditions.
+     * 
+     * @return the list of messages
+     */
+    public List<String> getAdditionalMessages() {
+        return additionalMessages;
+    }
+
+    /**
+     * A simplified string representation of this validation result for debugging purposes.
+     * 
+     * @return the string representation of this validation result
+     */
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+        String joinedErrors  = StringUtils.join(errors.stream().map((e) -> e.name()).collect(Collectors.toList()), ", ");
+        String joinedConditionResults = StringUtils.join(
+            conditionResults.stream().map((r) -> r.toString()).collect(Collectors.toList()), 
+            ", "
+        );
+        String joinedMessages =  StringUtils.join(additionalMessages, ",");
+        builder.append("LtpValidationResult " + super.toString() + "\n");
+        builder.append("- state: " + state.name() + "\n");
+        builder.append("- errors: " + joinedErrors + "\n");
+        builder.append("- conditionResults: " + joinedConditionResults + "\n");
+        builder.append("- addtionalMessages: " + joinedMessages + "\n");
```

Comment:
In a StringBuilder, shouldn't you use plus, because that creates an inner StringBuilder and converts it to a string? And what is `super.toString()` doing here? And isn't the line break platform-dependent? I think it should be, but check it yourself:
```suggestion
        builder.append("LtpValidationResult").append(System.lineSeparator());
        builder.append("- state: ").append(state.name()).append(System.lineSeparator());
        builder.append("- errors: ").append(joinedErrors).append(System.lineSeparator());
        builder.append("- conditionResults: ").append(joinedConditionResults).append(System.lineSeparator());
        builder.append("- addtionalMessages: ").append(joinedMessages);
```

This doesn't have to be changed, but it's a thought.